packages/datagateway-download/cypress/integration/downloadCart.spec.ts

```diff
@@ -137,7 +137,8 @@ describe('Download Cart', () => {
     );
   });
 
-  it.only('should be able open and close the download confirmation dialog', () => {
+  // TODO: only?
```

Comment:
If these tests work, remove comment and keep change from `it.only` to `it`. Otherwise, check why we were using `only` here.

Reply:
@patrick-austin I think this should not have been `only`, it seems it has been there maybe from testing.